src/include/rbd/librbd.hpp

```diff
@@ -38,6 +39,7 @@ namespace librbd {
     uint64_t id;
     uint64_t size;
     std::string name;
+    utime_t stamp;
```

Comment:
yes... So in this way, a new struct should also be made,which contains snapshot id, and timestamp. Is it ok for yiu?